terraform/modules/api-gateway-to-s3/auth.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_iam_policy" "s3_policy" {
+  name        = "s3-policy"
+  tags        = {}
```

Comment:
Can we add some basic tagging to our resources? `team`, `project`, `environment` perhaps?

Other teams are going to be using this account also - so let's start with some decent hygiene around our stuff

Reply:
I agree! I've added these default tags as you suggested

```
provider "aws" {
  profile = "default"
  region  = "us-east-2"
  default_tags {
    tags = {
      Team        = "NearForm"
      Project     = "AWS-IPFS"
      Environment = "POC"
    }
  }
}
```